src/oss/confluent.sh

```diff
@@ -1433,6 +1479,65 @@ version_command() {
     fi
 }
 
+produce_usage() {
+    local exit_status="${1}"
+    if [[ -z "${exit_status}" ]]; then
+      exit_status=0
+    fi
+
+    cat <<EOF
+Usage: ${command_name} <topicname> [--value-format avro] [other optional args]
```

Comment:
I don't think we should say `confluent produce` supports optional args. I'd prefer if we didn't commit to support the optional args of the underlying script. We can say we support a simple `produce` mode and list all the expected args explicitly (only topicname here and with the avro switch the expected properties such as key.schema and value.schema). We could still be lenient and not check the argument list. But we wouldn't commit to any special argument being supported.

Reply:
Why not?  Confluent provides the command line bin scripts, and this is just a straight pass-thru.